### DISCUSSION TITLE
Croft Manor

### DISCUSSION THREAD

**Discussion by F0mx** [nzp-team/nzportable#963] [Map Releases]
# CROFT MANOR

## _Despite all of her adventures, nothing could have prepared Lara for the horrors of element 115... And now, zombies on her doorstep, she is fighting for survival once again..._

The iconic home level from Tomb Raider 2 reimagined as an NZP map. Borrows some elements from the third installments home level to bring a more cohesive overall experience.

#  [DOWNLOAD -  Version 1.1.0](https://archive.org/download/croft-manor/Croft_Manor.zip)

### Features: 
- Kino-style teleporter room with Pack-a-punch
- Randomized Perk locations
- Randomized Box locations
- Perk machine jingles
- 5 Teddy bears to find; 
  - Find all 5 for a buyable secret Easter Egg weapon & Song
- 3 Dragon trophies to find;
  - Find all 3 for a buyable secret Easter Egg weapon & Song
- Multiple Minor Easter Eggs to find including a third song
- Full Easter Egg with map ending

### Screenshots: 
<img width="2560" height="1377" alt="fte-20260815152641-0" src="https://github.com/user-attachments/assets/c38e6e7a-a483-4865-bb9f-c9fd4eb5417c" />
<img width="2560" height="1377" alt="fte-20260815152722-0" src="https://github.com/user-attachments/assets/bf7b5189-1d1b-42de-9507-7ffc99625e5a" />
<img width="2560" height="1377" alt="fte-20260815152948-0" src="https://github.com/user-attachments/assets/74e397dc-00f2-4b2c-be6e-3037aa4375c5" />
<img width="2560" height="1377" alt="fte-20260815153011-0" src="https://github.com/user-attachments/assets/e3c43300-2bab-4c62-beec-7e99c81c8d03" />
<img width="2560" height="1377" alt="fte-20260815153024-0" src="https://github.com/user-attachments/assets/80655291-f138-4bc9-a364-4d928ad3586d" />

### Credits: 
Big thank you to Cypress and DerpedCrusader for all their help with bringing this map to life, this was my first attempt at making a map (I wont disclose how many hours I have put into it cause its kinda gross) and they were both a massive help in getting it to where it is today. 

# _Changelog:_

_17/08/2026_
_Version 1.1.0 - Full Release_
_- Unified release! Now works on all platforms using a single version._
_- Added Zoning to greatly improve map flow._
_- Additional spawns added to the kitchen._
_- Full lighting overhaul to improve visuals and visibility on all platforms._
_- Added loading screen music._
_- Fixed the buyable ending._
_- Simplified perk spawn locations. Juggernog and Speed-Cola will spawn between the Freezer and the Pool area._
_- Full waypointing overhaul, pathfinding should no longer have issues._
_- Optimization pass for better performance on all platforms._
_- Fixed some broken message strings to now display properly on all platforms._

_15/11/2024_
_Version 0.17.4.1 - 3DS/PSP ONLY_
_- Added dedicated files for 3DS/PSP versions_

_11/11/2024_
_Version 0.17.4_
_- Added perk machine jingles / stings_
_- Added PaP sting on use_
_- Added ambient music to the Aquarium_
_- Filled out the Aquarium with fish and other details_
_- Made all weapon walls illusory, most notable for making the scoped kar98k easier to purchase_
_- Replaced a lot of temporary assets, namely debris piles_
_- Replaced / fixed a lot of models & externalised their textures_
_- Overhauled the pool area, significantly improved pathing_ 
_- Speculative fix for the random perk crashing issue due to jug not having a place to go when all other spots were taken_
_- Minor optimizations_

_08/11/2024_
_Version 0.17.2_
_- Optimized skybox and lighting_
_- Changed file naming scheme to be more easily copy pasting and allow use on other platforms_
_- Speculative fix for teleporting zombies from the bedroom spawns_
_- Improved level geometry and waypointing in pool area_
_- Added more waypoints outside the manor_
_- Fixed the bathtub geometry to allow zombies out that spawn there_
_- Fixed the kitchen door being purchasable_
_- Fixed the door between the dining hall and they gym not activating respective spawn points_

_07/11/2024_
_Version 0.17.1 - Initial Release._

**Comment by TreeTop72** [comment]
Actually one of the most underrated maps I've ever played, if this was made on BO3 it would be amazing

**Comment by MisaelJama** [comment]
Please could you port it to psp please I beg you🥺

**Reply by F0mx** [reply]
I am working on getting a copy running on 3DS  / PSP, currently at the map booting without any custom models working but doors are also broken

**Reply by F0mx** [reply]
Added a working version now, be warned its visually buggy and has some pathfinding issues

**Reply by MisaelJama** [reply]
Thank you very much for the map you made for psp thanks you made me happy with that map🥺❤️

**Comment by tottalynotevil34332** [comment]
i love it looks like my old house

**Comment by MisaelJama** [comment]
Dude are you working on another map because your map was amazing and I can't wait to try another one

**Comment by MisaelJama** [comment]
What do I have to do to be able to open the door to go outside because I have already collected the amount that they are asking for and it still doesn't open for me?

**Reply by F0mx** [reply]
If you mean the doors that cost -31073, they're placeholder doors that are intentionally unopenable

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard because you have different downloads depending on the platform and the builds are not hosted on archive.org.

**Reply by InfusedStormblessed** [reply]
Sorry for bothering you, but why does everything have to be onarchive.org?

**Comment by Fiilthy** [comment]
For anyone else running into light flickering on 3DS, adjust _rl_fullbright_ value to "1" in console.

**Reply by F0mx** [reply]
Where abouts are you running into flickering? I hadn't run into any in my few playthroughs

**Reply by katniny** [reply]
> Where abouts are you running into flickering? I hadn't run into any in my few playthroughs

it could possibly be related to the lightmap issue? since there's a known one for 3DS and they did say, "For anyone else running into light flickering on 3DS ..."

**Comment by tayyabkhan21** [comment]
pls tell me where the teddy bears are. im too lazyyy

**Reply by F0mx** [reply]
Nah

**Comment by tayyabkhan21** [comment]
also, when i try to shoot the dragon trophies, nothing happens. why?

**Reply by F0mx** [reply]
They're intractable, not shootable

**Comment by nonMat** [comment]
This map is amazing! 
I did about everything you can in the map but I couldn't find the ending )':
Still this is great! If you ever update it I'll definitly check it out!
Now to beat it in gun game!

**Reply by F0mx** [reply]
The weapons board in the PaP room is key to finding the ending ;)

**Reply by nonMat** [reply]
Idk I got every weapon and also got outside the mansion but then I couldn't find anything else...

**Reply by F0mx** [reply]
Yeah I have had a few recent reports of that, this map is due a full release at some point soonTM
I'll need to take a look at that

**Reply by DinoDorian** [reply]
> Yeah I have had a few recent reports of that, this map is due a full release at some point soonTM I'll need to take a look at that

Hello F0mx! I am trying to beat this map and I cannot find the 5th teddy or any of the dragon trophies.
I found the teddy in the fireplace, the one on the chandelier, the one at pack-a-punch and the one near the bathroom. I do not know where the 5th one is

I also cannot find the dragons, this map is lots of fun and i want to beat it, please help.

**Reply by 420faded420** [reply]
last bear is in the teleporter room on top of the teleporter.

**Comment by F0mx** [comment]
Full release is finally here! I have gone through a fixed a ton of issues on the map as well as adding new features such as Zoning and loading screen music. Please let me know of any issues with the map that you run in  to or feedback in general! Hope y'all enjoy this new version as it has been a long time coming! ^^

17/08/2026
Version 1.1.0 - Full Release
_- Unified release! Now works on all platforms using a single version._
_- Added Zoning to greatly improve map flow._
_- Additional spawns added to the kitchen._
_- Full lighting overhaul to improve visuals and visibility on all platforms._
_- Added loading screen music._
_- Fixed the buyable ending._
_- Simplified perk spawn locations. Juggernog and Speed-Cola will spawn between the Freezer and the Pool area._
_- Full waypointing overhaul, pathfinding should no longer have issues._
_- Optimization pass for better performance on all platforms._
_- Fixed some broken message strings to now display properly on all platforms._

Known Issues:
_- 3DS hard crashes upon any musical easter egg being loaded._ #1504 
_- 3DS Sprites render through models._ #1505 
_- VRIL Skybox misalignment._  #1506
_- VRIL Sprites not affected by scale._ #1508 
_- VRIL Sprites rendering as fullbright._ #1507
_- VRIL Lightmap rotation issues._ #1509

**Reply by MLY22** [reply]
Amazing, I've played the map yesterday and it was great, but couldn't complete it yet

**Comment by nonMat** [comment]
This update is amazing!!!
Everything is so polished and well defined! (glad to see that all the windows have been blocked off!)
The only problem I found was in the ending area, where you can get out of bounds as shown in the clip below:

https://github.com/user-attachments/assets/d3062515-6198-4236-9595-fb5e7a73dccd

Either way great job!!!! It's so much fun!!!

**Reply by F0mx** [reply]
Whoops haha yeah I'll get that fixed up
Glad you're enjoying the updated version ^^

**Comment by M0THRAK** [comment]
Not a big issue, but it appears some of the models don't have transparency? fyi I'm playing this map on FTE.

<img width="640" height="320" alt="image" src="https://github.com/user-attachments/assets/8c5e15a3-d6d3-4f88-be57-17a60ec8ead3" />

**Comment by 420faded420** [comment]
played this map alot before the update.the new update makes it even better then it was before,love the new lighting
